app/src/main/java/info/anwesha/iitp/team/TeamFragment.java

```diff
@@ -36,25 +37,18 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
-        //allMembers.add(new info.anwesha.iitp.team.TeamItem("Piyush Tiwari", "Coordinator - Events and Operations", "9834943057", "https://www.facebook.com/thebackpropogator", R.drawable.piytwr_c1));
-       // allMembers.add(new info.anwesha.iitp.team.TeamItem("Mohit Kishore", "Coordinator - Events and Operations", "9570566557", "https://www.facebook.com/originalmk7", R.drawable.mohit_c1));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Vatsal Singhal", "Coordinator - Development Team", "8585992062", "https://m.facebook.com/vatsalsinghal1", R.drawable.vatsal_singhal));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Kshitij Jain", "Coordinator - Development Team", "9131604185", "https://www.facebook.com/kshitij.jain.39904", R.drawable.kshitij_jain));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Adarsh Chaudhary ", "Coordinator - Development Team", "8827275145", "https://www.facebook.com/adarsh217", R.drawable.adarsh_chaudhary));
-       // allMembers.add(new info.anwesha.iitp.team.TeamItem("Srikar Nayak", "Coordinator - Creatives And Design", "9666663764", "https://www.facebook.com/srikar.nayak.12", R.drawable.srikar_c1));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Rajeshwar Ojha", "Coordinator - Creatives And Design", "07000125484", "https://www.facebook.com/rajeshwar.ojha.50", R.drawable.rajeshwar));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Utkarsh", "Coordinator - Creatives And Design", "08987044093", "https://www.facebook.com/utkarsh.virtuous", R.drawable.utkarsh));
-        //allMembers.add(new info.anwesha.iitp.team.TeamItem("Prateek Rai", "Coordinator - Management And Planning", "7222998383", "https://www.facebook.com/profile.php?id=100019405147477", R.drawable.prai_c));
-        //allMembers.add(new info.anwesha.iitp.team.TeamItem("Yashwanth Chowdhary", "Coordinator - Management And Planning", "9182994302", "https://www.facebook.com/yaswanthchowdary.muppalla.7", R.drawable.yashwanth_c1));
-        //allMembers.add(new info.anwesha.iitp.team.TeamItem("Rama Krishna", "Coordinator - Management And Planning", "8985265942", "https://www.facebook.com/ramakrishna.rapelly", R.drawable.rama_c1));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Rahul Grover", "Coordinator - Hospitality", "8847433272", "https://m.facebook.com/rahulgrover1999", R.drawable.grover));
         allMembers.add(new info.anwesha.iitp.team.TeamItem("Shivansh Shukla", "Coordinator - Hospitality", "8707499083", "https://facebook.com/shivansh.shukla.50", R.drawable.shivansh));
-        allMembers.add(new info.anwesha.iitp.team.TeamItem("Rakshit Bhatt", "Coordinator - Hospitality", "9627765739", "https://m.facebook.com/", rakshit_bhatt));
+        allMembers.add(new info.anwesha.iitp.team.TeamItem("Rakshit Bhatt", "Coordinator - Hospitality", "9627765739", "https://m.facebook.com/",rakshit_bhat));
```

Comment:
@Kavya-24 use R.drawable.rakshit_bhat here instead of rakshit_bhatt only. This will resolve the issue